sources/packages/web/src/components/generic/StatusBadge.vue

```diff
@@ -16,6 +16,7 @@ status Badge
 <script lang="ts">
 import { ref, onMounted, watch } from "vue";
 import { GeneralStatusForBadge } from "@/types";
+import "@/assets/css/global-style-variables.scss";
```

Comment:
why did we import global-style-variables here? I do not see an usage of sass variables

Reply:
removed